kani-driver/src/call_cbmc.rs

```diff
@@ -124,12 +124,17 @@ impl KaniSession {
             args.push("--validate-ssa-equation".into());
         }
 
-        if !self.args.visualize
-            && self.args.concrete_playback.is_none()
-            && !self.args.no_slice_formula
-        {
-            args.push("--slice-formula".into());
-        }
+        // Push `--slice-formula` argument.
+        // Previously, this would happen if the condition below was satisfied:
+        // ```rust
+        // if !self.args.visualize
+        //   && self.args.concrete_playback.is_none()
+        //   && !self.args.no_slice_formula
+        // ```
+        // But for some reason, not pushing it causes a CBMC invariant violation
+        // since version 5.68.0.
+        // <https://github.com/model-checking/kani/issues/1810>
+        args.push("--slice-formula".into());
```

Comment:
Does this mess with concrete playback?

Reply:
Needs more investigation.

My understanding was that using `--slice-formula` with concrete playback may remove calls to `kani::any()` from the trace, which doesn't allow us to provide a counter-example (in that case, we just report a test couldn't be generated). But all tests we have for concrete playback and `--visualize` pass if we use `--slice-formula`.